src/researches/getPassiveVoice.js

```diff
@@ -1,110 +1,122 @@
-var getSentences = require( "../stringProcessing/getSentences.js" );
-var stripHTMLTags = require( "../stringProcessing/stripHTMLTags.js" ).stripFullTags;
-var getLanguage = require( "../helpers/getLanguage.js" );
-var Sentence = require( "../values/Sentence.js" );
+const getSentences = require( "../stringProcessing/getSentences.js" );
+const stripHTMLTags = require( "../stringProcessing/stripHTMLTags.js" ).stripFullTags;
+const getLanguage = require( "../helpers/getLanguage.js" );
+const Sentence = require( "../values/Sentence.js" );
 
-// Imports used for English, French and Spanish.
-var getSentencePartsDefault = require( "./passiveVoice/getSentenceParts.js" );
-var determinePassivesDefault = require( "./passiveVoice/determinePassives" );
+const forEach = require( "lodash/forEach" );
 
-// Imports used for German.
-var getSentencePartsGerman = require( "./german/passiveVoice/getSentenceParts.js" );
-var determinePassivesGerman = require( "./german/passiveVoice/determinePassives.js" );
+const determinePassiveSentencePart = require( "./passiveVoice/periphrastic/determinePassiveSentencePart.js" );
```

Comment:
Would it make sense to have an `index.js` in the `periphrastic` folder which exports an object containing all the relevant functions.

So usage would become this:

```js
const periphrastic = require( "./passiveVoice/periphrastic" );
periphrastic.determineSentencePart();

// Or in the newer ES6 syntax:
import { determineSentencePart } from "./passiveVoice/periphrastic";
determineSentencePart();
```

Reply:
I would agree had there been many files to export. But if I understand correctly, we are talking about 2 (or potentially 4) lines of code. Can you explain your reasoning in more detail?